docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md

```diff
@@ -0,0 +1,30 @@
+# End of support for legacy manual organizations November 2, 2021
+
+On November 2, 2021, as part of our efforts to improve Codacy Cloud and allow all our customers to benefit from a more seamless experience, we’re removing the support for legacy manual organizations. As part of this process, we'll automatically migrate your legacy organization to a new [synced organization](../../organizations/what-are-synced-organizations.md).
+
+To make sure that Codacy will continue to analyze your repositories please perform the following steps before November 2, 2021:
+
+1.  **If you're using GitHub** you must [install the Codacy GitHub App](https://github.com/apps/codacy-production/installations/new) on the GitHub organizations that contain your repositories so that Codacy has the [necessary permissions](../../getting-started/which-permissions-does-codacy-need-from-my-account.md) to analyze your code.
+
+1.  Make sure that your Git provider organization owners [belong to the **Administrators** team](../../organizations/manual-organizations/creating-and-managing-teams.md) of your legacy organization on Codacy.
+
+If you already have any synced organizations on Codacy, the migration process won't overwrite any data on those synced organizations, nor copy any repositories that you have in your legacy organization.
+
+!!! important
+    After the migration process ends on December 2, 2021, we'll delete any existing legacy organizations that we couldn't migrate.
```

Comment:
Confirm that this is the date when we plan to finish the migration process.

Reply:
Confirmed [here](https://docs.google.com/document/d/1JaBzT-sjc-hstgF1B5QOf_cLD-AN5ifFCGeSksdnQEg/edit?disco=AAAAPTO3ohs).